### PR TITLE
adding shopify integration docs

### PR DESCRIPTION
**Pull request by hhargreaveseppo**

### CODE REVIEW

**Comment by bertilhatt** [comment]
Should this be listed under "Marketing integration"?

**Review by bertilhatt — APPROVED** [review]
Looks great. Included minor comments.

File: docs/guides/marketing/integrating-with-shopify.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+              );      
+          })
+          .catch((error) => {
+            console.error('Eppo SDK initialization failed:', error);
+          });
+      });
+    </script>
+```
+    
+- Replace `'<YOUR-FLAG-KEY>'` with the specific key you've configured in Eppo for the feature.
```
[the remark this answers is not in the document]
Comment (hhargreaveseppo) [reply docs/guides/marketing/integrating-with-shopify.md:70]: The user id for Shopify could be a whole topic. Shopify provides some anon ids through cookies for users that use Shopify analytics, if a user is logged in through Shop the store owner will have access to a logged in customer id. I typically see users use an id from their analytics platform (ie: Heap, Mixpanel, etc) to keep the id and therefore variation consistent. I think it's worth it to expand on this section in V2 of this integration guide. For now, I'll make it more clear what the purpose of the subject id is

File: docs/guides/marketing/integrating-with-shopify.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+- Since Shopify classic is not a single page app and reloads every page, you can set the `maxCacheAgeSeconds` to an interval to ensure a wait-time before fetching a fresh configuration. The Eppo client instance will reference the cached configuration in local storage until it is time to fetch a fresh configuration. You read more about the advanced initialization options [here](/sdks/client-sdks/javascript/#initialization-options).
```
[the remark this answers is not in the document]
Comment (hhargreaveseppo) [reply docs/guides/marketing/integrating-with-shopify.md:71]: Thanks for providing clearer language! I'll use this instead

**Pull request merged**